src/actions/arrow_navigation.ts

```diff
@@ -91,7 +93,9 @@ export class ArrowNavigation {
       e: Event,
       shortcut: ShortcutRegistry.KeyboardShortcut,
     ): boolean => {
-      const toolbox = workspace.getToolbox() as Toolbox;
+      const toolbox = workspace.isFlyout
+        ? workspace.targetWorkspace?.getToolbox()
+        : (workspace.getToolbox() as Toolbox);
```

Comment:
nit: do you need this cast? if you don't need it on the line above I don't know why you'd need it here. seems like it should be both or neither.

Reply:
Good catch, indeed it's not necessary. Removed!

Reply:
There's a couple other places in the same file you have the same cast which is probably unnecessary there too then